[css-values-5] Rename CSS URL modifiers to include hyphens

nt1m opened the pull request:
#### 4eb61ecb9e3c3e01c8bbdfe038b2b70464324bbe
<pre>
[css-values-5] Rename CSS URL modifiers to include hyphens
<a href="https://bugs.webkit.org/show_bug.cgi?id=296353">https://bugs.webkit.org/show_bug.cgi?id=296353</a>
<a href="https://rdar.apple.com/156449679">rdar://156449679</a>

Reviewed by Sam Weinig.

Spec change: <a href="https://github.com/w3c/csswg-drafts/commit/e1bf92d49ad8c5e8e4522d9838911e64a962e817">https://github.com/w3c/csswg-drafts/commit/e1bf92d49ad8c5e8e4522d9838911e64a962e817</a>

* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous-negative.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous-negative.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous-negative.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous-negative.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials-negative.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials-negative.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials-negative.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials-negative.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrer-policy.py: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrerpolicy.py.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrer-policy.sub.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrerpolicy.sub.js.
(make_test_url):
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-computed.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-computed.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-invalid.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-invalid.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-serialize.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-serialize.sub.html:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-computed.sub-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-invalid.sub-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-serialize.sub-expected.txt:
* Source/WebCore/css/CSSValueKeywords.in:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+URL.cpp:
(WebCore::CSSPropertyParserHelpers::consumeURLRaw):
* Source/WebCore/css/values/primitives/CSSURL.cpp:
(WebCore::CSS::Serialize&lt;URL&gt;::operator):
* Source/WebCore/css/values/primitives/CSSURLModifiers.cpp:
(WebCore::CSS::applyModifiersToLoaderOptions):
* Source/WebCore/css/values/primitives/CSSURLModifiers.h:
(WebCore::CSS::get):
* Source/WebCore/style/values/primitives/StyleURL.cpp:
(WebCore::Style::operator&lt;&lt;):

Canonical link: <a href="https://commits.webkit.org/297960@main">https://commits.webkit.org/297960@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e128cc0f8bb79b8e9926fa0e66822ccd3805fb93

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/113677 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/131/builds/33381 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/23821 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/119853 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/64453 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/115566 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/33989 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/41946 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/119853 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/64453 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/116625 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/27120 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/102143 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/119853 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/26359 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/20267 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/63570 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/96502 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/20370 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/123075 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 vision~~](https://ews-build.webkit.org/#/builders/128/builds/40677 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/30354 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/123075 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 vision-sim~~](https://ews-build.webkit.org/#/builders/121/builds/41068 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/98358 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/123075 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24243 "Built successfully and passed tests") | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/17981 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/36892 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/127/builds/40559 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-safer-cpp~~](https://ews-build.webkit.org/#/builders/120/builds/46068 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/125/builds/40206 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/43507 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/41974 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/65e48837ae64e8af1a2da7649d1875b18e4b6204. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/7cf0facfdcf95c91b35b4172294416d44f8b94b2. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/54617e31449891ffd310a01fb49536e1e39f3b98. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/1f2312a9fffa2933f3ef6e1264dd819bf8c6c1ce. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/808a8e96a9f15418f22c0ce61fabc1a0a72ff2d8. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/808a8e96a9f15418f22c0ce61fabc1a0a72ff2d8. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8d0febddf47f3965e4695ed0fc54dd4d17bb9ea1. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8d0febddf47f3965e4695ed0fc54dd4d17bb9ea1. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b3c914b6224dd377f5ae0ce5dfba766e67f179bb. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b3c914b6224dd377f5ae0ce5dfba766e67f179bb. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e128cc0f8bb79b8e9926fa0e66822ccd3805fb93. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e128cc0f8bb79b8e9926fa0e66822ccd3805fb93. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48398

nt1m committed:
[css-values-5] Rename CSS URL modifiers to include hyphens
https://bugs.webkit.org/show_bug.cgi?id=296353
rdar://156449679

Reviewed by Sam Weinig.

Spec change: https://github.com/w3c/csswg-drafts/commit/e1bf92d49ad8c5e8e4522d9838911e64a962e817

* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous-negative.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous-negative.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous-negative.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous-negative.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-anonymous.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-anonymous.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials-negative.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials-negative.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials-negative.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials-negative.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-cross-origin-use-credentials.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/url-image-crossorigin-use-credentials.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/cross-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer-when-downgrade/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/no-referrer/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin-when-cross-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/same-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin-when-cross-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/strict-origin/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-cross-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-cross-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-cross-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-cross-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-same-origin.sub-expected.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-same-origin.sub-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrer-policy-same-origin.sub.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/url-image-referrerpolicy-same-origin.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/referrer-policy/unsafe-url/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrer-policy.py: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrerpolicy.py.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrer-policy.sub.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/image-referrerpolicy.sub.js.
(make_test_url):
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-computed.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-computed.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-invalid.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-invalid.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-serialize.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-serialize.sub.html:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-computed.sub-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-invalid.sub-expected.txt:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-values/urls/url-request-modifiers-serialize.sub-expected.txt:
* Source/WebCore/css/CSSValueKeywords.in:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+URL.cpp:
(WebCore::CSSPropertyParserHelpers::consumeURLRaw):
* Source/WebCore/css/values/primitives/CSSURL.cpp:
(WebCore::CSS::Serialize<URL>::operator):
* Source/WebCore/css/values/primitives/CSSURLModifiers.cpp:
(WebCore::CSS::applyModifiersToLoaderOptions):
* Source/WebCore/css/values/primitives/CSSURLModifiers.h:
(WebCore::CSS::get):
* Source/WebCore/style/values/primitives/StyleURL.cpp:
(WebCore::Style::operator<<):

Canonical link: https://commits.webkit.org/297960@main

webkit-commit-queue commented:
Committed 297960@main (4eb61ecb9e3c): <https://commits.webkit.org/297960@main>

Reviewed commits have been landed. Closing PR #48398 and removing active labels.
